tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+from utils import produce_jwt
+from Crypto.PublicKey import RSA
+import os
+import jwt
+
+
+class Test_utils:
+
+    payload = None
+
+    def set_token(self):
+        if not self.payload:
+            with open(os.getenv('swag_pem'), "r") as f:
+                private_pem = f.read()
+            publickey = RSA.importKey(private_pem).publickey().exportKey('PEM')
+            jwtoken = produce_jwt()
+            self.payload = jwt.decode(jwtoken, publickey, algorithms='RS256')
+        return self.payload
+
+    def test_jwt_time(self):
+        payload = self.set_token()
+        assert payload['exp'] == payload['iat'] + 10 * 60
+
+    def test_jwt_appid(self):

```

Comment:
I don't think your tests check the actual functioning of payload making, maybe a test could be where you mock times and then compare it to an actual result. I can probably supply the token for it

Reply:
I added tests to test the accuracy of the issuing and expiration times of the JWT.